.gitignore

```diff
@@ -37,6 +37,10 @@ msbuild.log
 *.sln.ide
 *.ide/
 
+# jitutils
```

Comment:
I don't think we want this. These directories should get deleted doing `git clean -dxf`, for example. I don't know why you would want them to stick around. They're only created during the jitformat job, right?